src/pages/studyView/styles.scss

```diff
@@ -106,3 +106,16 @@ iframe.mdacc-heatmap-iframe {
     height: calc(100vh - 200px);
     min-height: 700px;
 }
+
+.resourcesTab {
+    .resourcesSection {
+        padding: 10px;
+        border-radius: 5px;
+        border: 1px dashed #ddd;
```

Comment:
lets try to accomplish this with bordered chart.   If we don't want to call it bordered chart then we can use some sass extend/inherit sorcery to apply this styling to both border-chart and this.  see sass basics

Reply:
good idea. see my new addition mixins.scss